dll/win32/advapi32/sec/lsa.c

```diff
@@ -29,10 +29,11 @@ LsapIsLocalComputer(PLSA_UNICODE_STRING ServerName)
 
     buf = HeapAlloc(GetProcessHeap(), 0, dwSize * sizeof(WCHAR));
     Result = GetComputerNameW(buf, &dwSize);
-    if (Result && (ServerName->Buffer[0] == '\\') && (ServerName->Buffer[1] == '\\')) {
-		Result = Result && !lstrcmpW((LPCWSTR)&ServerName->Buffer[2], buf); 
-	} else {
-		Result = Result && !lstrcmpW(ServerName->Buffer, buf); 
+    if (Result) {
+        if (ServerName->Buffer[0] == '\\' && ServerName->Buffer[1] == '\\')
+	    	Result = !lstrcmpW((LPCWSTR)&ServerName->Buffer[2], buf); 
+    	else
+	    	Result = !lstrcmpW(ServerName->Buffer, buf); 
 	} 
```

Comment:
I think the diff here could be reduced significantly if you introduce a new LPWSTR variable, assign the buffer to it, and then operate just on it.